src/ce/ti/hooks.h

```diff
@@ -0,0 +1,47 @@
+/**
+ * @file
+ * @authors
+ * Matt "MateoConLechuga" Waltz\n
+ * Jacob "jacobly" Young
+ * @brief TI CE OS hooks.
+ * The API for using hooks isn't fleshed out here, so this isn't really useful.
```

Comment:
Then why does this file exist

Reply:
Well somebody put those equates in `tice.h` and they definitely don't belong anywhere else. Maybe we should leave the file out of the documentation tree?

Reply:
File deteled, now hooks are @commandblockguy's problem.